### ISSUE TITLE
Error compiling ipt-netflow v2.4 on CentOS Linux release 7.7.1908

### ISSUE THREAD

**Issue by nklock** [aabc/ipt-netflow#131]
Hello,

I am attempting to compile ipt-netflow on CentOS Linux release 7.7.1908 however I am encountering a number of redefinition errors while compiling:

`[root@localhost ipt-netflow]# make all install
Compiling for kernel 3.10.0-1062.12.1.el7.x86_64
make -C /lib/modules/3.10.0-1062.12.1.el7.x86_64/build M=/root/ipt-netflow modules CONFIG_DEBUG_INFO=y
make[1]: Entering directory `/usr/src/kernels/3.10.0-1062.12.1.el7.x86_64'
  CC [M]  /root/ipt-netflow/ipt_NETFLOW.o
In file included from /root/ipt-netflow/ipt_NETFLOW.c:76:0:
/root/ipt-netflow/compat.h:619:24: error: redefinition of ‘xt_family’
 static inline u_int8_t xt_family(const struct xt_action_param *par)
                        ^
In file included from include/uapi/linux/netfilter_ipv4/ip_tables.h:22:0,
                 from include/linux/netfilter_ipv4/ip_tables.h:23,
                 from /root/ipt-netflow/ipt_NETFLOW.c:52:
include/linux/netfilter/x_tables.h:77:24: note: previous definition of ‘xt_family’ was here
 static inline u_int8_t xt_family(const struct xt_action_param *par)
                        ^
In file included from /root/ipt-netflow/ipt_NETFLOW.c:76:0:
/root/ipt-netflow/compat.h:623:40: error: redefinition of ‘xt_in’
 static inline const struct net_device *xt_in(const struct xt_action_param *par)
                                        ^
In file included from include/uapi/linux/netfilter_ipv4/ip_tables.h:22:0,
                 from include/linux/netfilter_ipv4/ip_tables.h:23,
                 from /root/ipt-netflow/ipt_NETFLOW.c:52:
include/linux/netfilter/x_tables.h:52:40: note: previous definition of ‘xt_in’ was here
 static inline const struct net_device *xt_in(const struct xt_action_param *par)
                                        ^
In file included from /root/ipt-netflow/ipt_NETFLOW.c:76:0:
/root/ipt-netflow/compat.h:627:40: error: redefinition of ‘xt_out’
 static inline const struct net_device *xt_out(const struct xt_action_param *par)
                                        ^
In file included from include/uapi/linux/netfilter_ipv4/ip_tables.h:22:0,
                 from include/linux/netfilter_ipv4/ip_tables.h:23,
                 from /root/ipt-netflow/ipt_NETFLOW.c:52:
include/linux/netfilter/x_tables.h:62:40: note: previous definition of ‘xt_out’ was here
 static inline const struct net_device *xt_out(const struct xt_action_param *par)
                                        ^
In file included from /root/ipt-netflow/ipt_NETFLOW.c:76:0:
/root/ipt-netflow/compat.h:631:28: error: redefinition of ‘xt_hooknum’
 static inline unsigned int xt_hooknum(const struct xt_action_param *par)
                            ^
In file included from include/uapi/linux/netfilter_ipv4/ip_tables.h:22:0,
                 from include/linux/netfilter_ipv4/ip_tables.h:23,
                 from /root/ipt-netflow/ipt_NETFLOW.c:52:
include/linux/netfilter/x_tables.h:72:28: note: previous definition of ‘xt_hooknum’ was here
 static inline unsigned int xt_hooknum(const struct xt_action_param *par)
                            ^
make[2]: *** [/root/ipt-netflow/ipt_NETFLOW.o] Error 1
make[1]: *** [_module_/root/ipt-netflow] Error 2
make[1]: Leaving directory `/usr/src/kernels/3.10.0-1062.12.1.el7.x86_64'
make: *** [ipt_NETFLOW.ko] Error 2`

This appears to be fixed in the master branch as it compiles without issue. Would it be possible to backport the fix to v2.4, or is a new release coming with the fix included?

**Comment by aabc** [comment]
Why can't you use master branch?

**Comment by nklock** [comment]
I'm just trying to follow best practices in my deployments, typically the master branch is considered "unstable" where as the tagged versions are supposed to be the stable releases.On Feb 21, 2020 5:14 AM, ABC <notifications@github.com> wrote:Why can't you use master branch?

**Comment by aabc** [comment]
Right. But I assure you there is only stability and compatibility improvements in master branch currently.
